catalog/admin/module/catalog-admin-module-sources/src/main/webapp/js/view/Utils.js

```diff
@@ -62,7 +62,7 @@ function ($,_, Marionette) {
          * Set up the popovers based on if the selector has a description.
          */
         setupPopOvers: function($popoverAnchor, id, title, description) {
-            var selector = ".description[data-title='" + id + "']",
+            var selector = ".description",
```

Comment:
why did it have the data-title attribute filter in the selector in the first place is there going to be other unintended elements that will get a popover now